### PR TITLE
WIP: added bot navigation, so far only source tag

### PR DESCRIPTION
**Pull request by dharm1k987**

### CODE REVIEW

File: src/components/sources/Sources.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              <div className="ba bw1 b shadow-4 br3 bg-white f5 blue mt3">
+              <Link to="/" className="flex justify-center">
+                <ReplyIcon className="mv2 mh2"/>
+              </Link>
+            </div>
+              <div className="flex flex-column items-center">
+                {
+                    listings.map(e => {
+                        return (
+                            <Card className="main-card mv2 ba b--light-silver bg-white br4" key={uuidv4()}>
```
Comment (cybervinit) [inline src/components/sources/Sources.jsx:65]: on the desktop, the Home button, title and description still much larger than the cards. Increase the card size to at least `w-50-ns` so there isn't as much empty space in desktop mode as right now

File: src/components/covid19-readiness-test/Covid19-readiness-test.jsx [hunk 2]
```diff
@@ -207,19 +208,21 @@ export class Covid19ReadinessTest extends Component {
...
                 <PlayArrowIcon className="mv2"/>
                 <div className="mv2 mh2">Start</div>
               </div>
             </button>
-            <HomeBtn />
+            <div className="ba bw1 b shadow-4 br3 bg-white f5 blue mt3">
+              <Link to="/" className="flex justify-center">
+                <ReplyIcon className="mv2 mh2"/>
+              </Link>
+            </div>
```
Comment (cybervinit) [inline src/components/covid19-readiness-test/Covid19-readiness-test.jsx:225]: Replace with `<HomeBtn />`

File: src/components/sources/Sources.jsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            
+              <h1 className="f2 tc pv4 b">COVID-19 Sources</h1>
+              <h2 className="tc">The following cards indicate which sources are used on this website to provide the most accurate COVID-19 information.
+              In each card is also listed how the information is being used, for your convenience.
+              </h2>
+              <div className="ba bw1 b shadow-4 br3 bg-white f5 blue mt3">
+              <Link to="/" className="flex justify-center">
+                <ReplyIcon className="mv2 mh2"/>
+              </Link>
+            </div>
```
Comment (cybervinit) [inline src/components/sources/Sources.jsx:60]: Replace with `<HomeBtn />`

File: src/components/covid19-readiness-test/Covid19-readiness-test.jsx (hunk 2 above)
Comment (dharm1k987) [reply src/components/covid19-readiness-test/Covid19-readiness-test.jsx]: Done

File: src/components/sources/Sources.jsx (hunk 3 above)
Comment (dharm1k987) [reply src/components/sources/Sources.jsx]: Done

File: src/components/sources/Sources.jsx (hunk 1 above)
Comment (dharm1k987) [reply src/components/sources/Sources.jsx]: tablet view is a little wonky because of jhu full form, but should be good for now

**Pull request merged**